package.json

```diff
@@ -20,31 +21,48 @@
   "private": false,
   "trackingCode": "UA-13244752-7",
   "engines": {
-    "node": ">= 8"
+    "node": ">= 12"
   },
   "keywords": [
     "movableink",
     "web crop",
     "email"
   ],
   "bin": {
-    "movable": "./bin/movable"
+    "movable": "./bin/movable",
+    "movable-beta": "beta/lib/cli.js"
   },
   "dependencies": {
+    "@movable-internal/generator-remote-upload-blueprint": "latest",
```

Comment:
Hmm, I think this will present issues, if we have a publicly available `movable` that depends on private npm modules. I think we'd either need to make all of the dependencies public, or maybe just consider creating a new, private tool?